projects/habitat_ovmm/metrics_utils.py

```diff
@@ -0,0 +1,107 @@
+from typing import Optional
```

Comment:
Merge main and move this file to `projects/habitat_ovmm/utils`.

Reply:
Done